testing/setconnectionstest.py

```diff
@@ -17,16 +16,18 @@ def testParsing(self):
                          (b"hello there", b"/goodbye:euoeu", None))
         self.assertEqual(pl(b"a:b:c:d::e"), (b"a:b:c:d", b"e", None))
         self.assertEqual(pl(b"foobar"), (None, b"foobar", None))
-        if os.name != "nt":  # FIXME must adapt escape handling for Windows
-            self.assertEqual(pl(rb"test\\ing\::more::and more\\.."),
-                             (rb"test\ing::more", rb"and more\\..", None))
-            self.assertEqual(pl(rb"hello\::there"),
-                             (None, rb"hello\::there", None))
-            self.assertEqual(pl(rb"foobar\\"), (None, rb"foobar\\", None))
+        self.assertEqual(pl(rb"test\\ing\::more::and more\\.."),
+                         (b"test\\ing::more", b"and more/..", None))
+        self.assertEqual(pl(rb"strangely named\::file"),
+                         (None, b"strangely named::file", None))
+        self.assertEqual(pl(rb"foobar\\"), (None, b"foobar/", None))
+        self.assertEqual(pl(rb"not\::too::many\\\::paths"),
+                         (b"not::too", b"many/::paths", None))
 
         # test missing path and missing host
-        self.assertIsNotNone(pl(rb"hello\:there::")[2])
```

Comment:
Why is this test not relevant anymore ? 
`hello\:there::`

Reply:
I just renamed the string to make the test more explicit, but it's basically the same test.